### ISSUE TITLE
Header not decoded

### ISSUE THREAD

**Issue by soywod** [dantecatalfamo/himalaya-emacs#2]
I'm not sure if it is related to your plugin, but it looks like headers (in my case `To` and `Subject`) are not decoded properly when reading emails. Recently, I received an email with those headers:

```eml
To: =?Windows-1252?Q?Cl=E9ment_DOUIN?= <clement.douin@posteo.net>
Subject: =?Windows-1252?Q?Re:_Prise_de_contact_=97_D=E9veloppement?=
```

**Comment by soywod** [comment]
Linked to https://github.com/soywod/himalaya/issues/140, I will take care of this whenever I can!

**Comment by soywod** [comment]
The feature has been added on `development`, now it is possible to insert headers with the `read` command: `himalaya read 42 -h subject -h to`. I will propose soon a PR to fix the emacs plugin!

**Comment by soywod** [comment]
Fixed by #3.
